app/javascript/analytics/dashboard.js

```diff
@@ -213,8 +215,8 @@ function renderReferrers(data) {
   if (emptyDomainReferrer) {
     tableBody.push(`
       <tr>
-        <td>All other external referrers</td>
-        <td>${emptyDomainReferrer.count}</td>
+        <td class="alignt-left">All other external referrers</td>
```

Comment:
```suggestion
        <td class="align-left">All other external referrers</td>
```

looks like a wee typo here - I think it's not having any negative affect though as all looks aligned OK to me (I guess 'left' is the default)

Reply:
This PR is a typo fest for me 😳  Thanks again for the catch!!